webpack.config.js

```diff
@@ -16,6 +16,13 @@ const CustomTemplatedPathPlugin = require( '@wordpress/custom-templated-path-web
 const LibraryExportDefaultPlugin = require( '@wordpress/library-export-default-webpack-plugin' );
 const { BundleAnalyzerPlugin } = require( 'webpack-bundle-analyzer' );
 
+/**
+ * Local dependencies
```

Comment:
Internal? Not sure if there's a distinction intended to be made.

Reply:
Yes, internal 👍